internal/view/content.html

```diff
@@ -38,6 +37,9 @@
 		<div id="content" dir="auto" v-pre>
 			$$.HTML$$
 		</div>
+        <div id="footer">
+            <p id=metadata> {{ createdtime() === modifidetime() ? `Added ` +  createdtime() : `Added ` + createdtime() + '  |  Last Modified ' + modifidetime() }} </p>
```

Comment:
Shouldn't something like this be more clear?

```suggestion
            <p id=metadata>Added {{ createdtime() }} {{ createdtime() !== modifidetime() ? `Added ` + createdtime() + '  |  Last Modified ' + modifidetime() }} </p>
```